core/dbt/events/types.py

```diff
@@ -746,6 +746,30 @@ def message(self) -> str:
         return f"Finished running {self.stat_line}{self.execution} ({self.execution_time:0.2f}s)."
 
 
+class ConstraintNotEnforced(WarnLevel):
+    def code(self):
+        return "E048"
+
+    def message(self) -> str:
+        msg = (
+            f"We noticed you have `{ self.constraint }` configs, these are NOT enforced "
+            "with this adapter and will be ignored"
+        )
+        return msg
+
+
+class ConstraintNotSupported(WarnLevel):
+    def code(self):
+        return "E049"
+
+    def message(self) -> str:
+        msg = (
+            f"We noticed you have `{ self.constraint }` configs, these are NOT supported "
```

Comment:
I think these messages were carried over from the initial implementation of model contracts. @dbeatty10 - curious to hear if you have any feedback on them!

Reply:
The first-person plural here, and the capitalized "NOT," feel out of keeping with our style in other warnings/error messages.

How about something like (total untested pseudo-code):
```python
# ConstraintNotEnforced
description = (
    f"The constraint type {constraint.type.value} is not enforced by {self.adapter_type}. "
    "The constraint will be included in this model's DDL statement, but it will not guarantee anything about the underlying data. "
    "Set 'warn_unenforced: false' on this constraint to ignore this warning."
)
msg = (line_wrap_message(warning_tag(description))
return msg
```

```python
# ConstraintNotSupported
description = (
    f"The constraint type {constraint.type.value} is not supported by {self.adapter_type}, and will be ignored. "
    "Set 'warn_unsupported: false' on this constraint to ignore this warning."
)
msg = (line_wrap_message(warning_tag(description))
return msg
```
That would require adding `{adapter.type()}` to these event definitions & call sites